src/renderer/src/App.tsx

```diff
@@ -0,0 +1,41 @@
+import { CssBaseline, ThemeProvider } from '@mui/material'
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
+import { RouterProvider, createRouter } from '@tanstack/react-router'
+
+import { theme } from './Theme'
+import {
+	ActiveProjectIdProvider,
+	CreateActiveProjectIdStore,
+} from './contexts/ActiveProjectIdProvider'
+import { ApiProvider } from './contexts/ApiContext'
+import { IntlProvider } from './contexts/IntlContext'
+import { routeTree } from './routeTree.gen'
+
+const queryClient = new QueryClient()
+
+const router = createRouter({ routeTree })
+
+declare module '@tanstack/react-router' {
+	interface Register {
+		router: typeof router
+	}
+}
+
+const PersistedProjectIdStore = CreateActiveProjectIdStore({
```

Comment:
```suggestion
const persistedProjectIdStore = createActiveProjectIdStore({
```

I think my comment before was maybe not very clear, I did not mean for you to change the case of CreateACtiveProjectIdStore